.github/workflows/partial-backend.yml

```diff
@@ -27,6 +27,12 @@ jobs:
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
         ports:
           - 5432:5432
+      ldap:
+        image: rroemhild/test-openldap
+        ports:
+          - 10389:10389
+          - 10636:10636
```

Comment:
```suggestion
```

You shouldn't have to expose any ports if you use the service name (similar to how you can do in docker-compose).